packages/liquidator/package.json

```diff
@@ -32,7 +32,7 @@
   "bin": "index.js",
   "scripts": {
     "test": "yarn mocha-test && yarn hardhat-test",
-    "hardhat-test": "hardhat test --network hardhat",
```

Comment:
this change was made as for some or other unknown reason having this `--network hardhat` broke the `ProviderUtils.js` with the following error:

![image](https://user-images.githubusercontent.com/12886084/106151099-ad89d200-6184-11eb-83a8-9bacbb9e4ebc.png)

we should consider removing this on all other packages as well but for now they seem to be passing CI.